src/usertiming.js

```diff
@@ -119,10 +119,11 @@
     var performanceTimelineRequiresSort = false;
 
     //
-    // If getEntries() isn't defined, we'll assume
-    // we have to shim all of the PT functions.
+    // If getEntries() and mark() aren't defined, we'll assume
+    // we have to shim at least some PT functions.
     //
-    if (typeof(window.performance.getEntries) !== 'function') {
+    if (typeof(window.performance.getEntries) !== 'function' ||
+        typeof(window.performance.mark) !== 'function') {
```

Comment:
@nicjansma I wasn't quite sure what'd be best here — I could just swap out `getEntries` for `mark` instead of doing two checks, but I don't know if there are browsers out there that might implement `mark` but *not* the getters. Seems unlikely, but there might be a slight risk of breaking some browser by doing that. What do you think?